examples/patterns/accordion.html

```diff
@@ -0,0 +1,175 @@
+---
+layout: default
+title: Accordion
+category: _patterns
+---
+
+<aside class="p-accordion" role="tablist" aria-multiselect="true">
+  <ul class="p-accordion__list">
+    <li class="p-accordion__group">
+      <button class="p-accordion__tab" id="owner-tab" role="tab" aria-controls="#owner" aria-expanded="true">Owner</button>
+      <section class="p-accordion__panel" id="owner" role="tabpanel" aria-hidden="false" aria-labelledby="owner-tab">
+        <p>Open panel</p>
+      </section>
+    </li>
+    <li class="p-accordion__group">
+      <button class="p-accordion__tab" id="status-tab" role="tab" aria-controls="#status" aria-expanded="false">Status</button>
+      <section class="p-accordion__panel" id="status" role="tabpanel" aria-hidden="true" aria-labelledby="status-tab">
+        <p>Closed panel</p>
+      </section>
+    </li>
+    <li class="p-accordion__group">
+      <button class="p-accordion__tab" id="tags-tab" role="tab" aria-controls="#tags" aria-expanded="false">Tags</button>
+      <section class="p-accordion__panel" id="tags" role="tabpanel" aria-hidden="true" aria-labelledby="tags-tab">
+        <p>Closed panel</p>
+      </section>
+    </li>
+  </ul>
+</aside>
+
+<h2>Example JavaScript</h2>
```

Comment:
Don't need this example js section. The example js displays the contents of the body so it will show it in the code snippet already.